wsimod/nodes/land.py

```diff
@@ -747,6 +831,39 @@ def __init__(
 
         self.outflows.append(self.route)
 
+    def apply_overrides(self, overrides = Dict[str, Any]):
+        """Override parameters.
+
+        Enables a user to override any of the following parameters: 
+        field_capacity, wilting_point, total_porosity, infiltration_capacity,
+        surface_coefficient, percolation_coefficient, et0_coefficient, ihacres_p,
+        soil_temp_w_prev, soil_temp_w_air, soil_temp_w_deep, soil_temp_deep,
+        and the corresponding parameter values, including field_capacity_m,
+        wilting_point_m, depth, capacity, subsurface_coefficient.
+
+        Args:
+            overrides (Dict[str, Any]): Dict describing which parameters should
+                be overridden (keys) and new values (values). Defaults to {}.
+        """
+        
+        self.depth /= self.total_porosity # restore the physical depth (root)

```

Comment:
Maybe `depth` should be a `@property`? Possibly is too linked with other things, not sure

Reply:
I think we need to try and test. I'll flag this in an issue and try to revise it when I get a bit more time. #95